draft-ietf-jsonpath-base.md

```diff
@@ -1264,6 +1264,32 @@ Queries:
 | `$[0, 0]` | `"a"` <br> `"a"` | `$[0]` <br> `$[0]` | Duplicated entries |
 {: title="List selector examples"}
 
+## Semantics of `null`
+
+JSON `null` is treated as a valid value, rather than "undefined" nor "missing."
```

Comment:
It's unclear what is meant by JSON null being a "valid" value, valid in what sense? as opposed to an "invalid" value?

This is about the semantics of the _JSONPath spec_, not the semantics of JSON `null`. For the example given of a query that doesn't evaluate to any results, `$.a[0]`, the spec writers could reasonably choose from several alternatives: (1) stay within the JSON type system, and have it evaluate to one of the JSON values, possibly `null` (like JMESPath), or even `false`, (2) extend the JSON type system with additional values such as `undefined` (following JavaScript), or `optional`, and have it evaluate to that, or (3) treat it as an error (like Jayway JSONPath).  None of these alternatives makes JSON `null` more valid or less valid.

For comparison, in original Goessner JsonPath, it is specified that JSONPath expressions return either an `array` or `false`, "false in case of no match."

Would you write "Unlike in original Goessner JsonPath, in IETF JsonPath `false` is treated as a valid value rather than undefined or missing"? I wouldn't think so.

Reply:
Yes.  See my comment above.
https://github.com/ietf-wg-jsonpath/draft-ietf-jsonpath-base/pull/181#discussion_r856064412

Reply:
Thanks @danielaparker and @cabo. This is fixed with an edited form of @cabo's suggestion.